SECURITY.md

```diff
@@ -12,3 +13,62 @@
 Please report any vulnerabilities via our [Bug Bounty Program](https://bugcrowd.com/owaspzap).
 
 Remote Code Execution for this program will be rewarded at __$1000__
+
+## Reporting a Bug
+
+Please report any bugs via our issue [Bug Issue Tracker](https://github.com/zaproxy/zaproxy/issues/new/choose)
+
+## Security Advisories
+
+Currently, there are not any published security advisories.
```

Comment:
I’m don’t think this is true/accurate, I’d have to go digging. Pretty sure we have/had at least one.

Reply:
We have had some security bugs reported which we've fixed, but we have not published ny security advisories. 
Maybe we should...

Reply:
I thought one or two had assigned CVEs (via the reporter not us), which I was lumping into the 'advisory' category.